### ISSUE TITLE
How did you manage to get craft to use the proejct.yaml file on install

### ISSUE THREAD

**Issue by terryupton** [madebyshape/craft-3#10]
Hey Jason.
I am trying to resolve my Craft boilerplate to use my `project.yaml` file to install plugins, setting various fields and sections and so forth on. However, whenever I run ./craft setup with my own boilerplate. It overwrites my `project.yaml` file rather than using it to build and setup Craft.

I have noticed when testing your setup, it uses the `project.yaml` correct on install/setup.
How did you achieve this?

**Comment by bymayo** [comment]
@terryupton I don't think we're doing anything too different to what you're doing. The `project.yaml` file just runs when we do `./craft setup`. I have noticed though sometimes this file will not run if there are errors in it (Usually gets logged).

**Comment by bymayo** [comment]
@terryupton Just following up on this. We had a similar issue recently, but when checking out the problem it seems Craft was throwing a few errors referring to the `.env` file, when running `./craft setup`. 

It seems that because we're using a custom `.env` file that we need to manually fill these details in before we install Craft. Particularly the `SITE_PATH` as this is what is used to create the Asset Volumes. But because it couldn't find the path, it wouldn't create the volume and just errored.

Once we do this we then run `./craft install` to generate a user.

**Comment by terryupton** [comment]
@bymayo - thanks for this. I had forgotten to reply previously.
I managed to get my `yaml` file working, it was a case of working through it and eradicating the errors. I think i had a few issues with some plugins have version numbers in the `yaml` that then didn't match when a fresh install occurs. I had a few other minor bugs too and I think one of these might have been setting in the `/.env` file.

Saying that I have not tested it again since I got it working. So hoping nothing has broken it again. I  should probably test it on a regular basis.
